src/app/code/community/Omise/Gateway/Model/Payment.php

```diff
@@ -6,17 +6,18 @@ abstract class Omise_Gateway_Model_Payment extends Mage_Payment_Model_Method_Abs
      */
     protected $_omise;
 
-    /**
-     * @var array
-     */
-    private $_currency_subunits = array(
-        'JPY' => 1,
-        'THB' => 100,
-        'SGD' => 100,
-        'USD' => 100,
-        'EUR' => 100,
-        'GBP' => 100
-    );
+    // /**
```

Comment:
please remove that comment

Reply:
5555... I already removed all these. Just looked at editor - I hadn't clicked save!! :P